### DISCUSSION TITLE
Notifiche Alexa verso device che fanno parte di gruppi

### DISCUSSION THREAD

**Discussion by varnone** [indomus/forum#1760] [HUB: Home Assistant]
Buongiorno a tutti.

Ho notato che riesco a mandare messaggi ad un dispositivo echo autonomo ma non riesco a mandare alcun messaggio a dispositivi che fanno parte di gruppi o al nome del gruppo stesso.

Qualcuno di voi è riuscito a mandare messaggi a device echo che sono parte di un gruppo ?

Grazie in anticipo.

Vincenzo

**Comment by Scialla** [comment]
i gruppi li importa alexa media player così come importa i singoli echo. Ad esempio io ho media_player.ovunque e, inviando un messaggio a questo, viene riprodotto su tutti

**Reply by varnone** [reply]
Confermo che io vedo il 100% dei dispositivi sia singoli che gruppo ma il comando di invio che funziona verso il dispositivo singolo non funziona verso il gruppo
Ad esempio questo funziona

`service: notify.alexa_media_cucina
data:
  message: Buongiorno sono Vincenzo
  title: prova
  data:
    type: tts`

Questo non funziona

`service: notify.alexa_media_camera_matrimoniale
data:
  message: Ciao sono Vincenzo
  data:
    type: tts`

Da notare che sul dispositivo di gruppo camera_matrimoniale sono in grado di regolare il volume dei dispositivi ma non sono in grado di fare pronunciare notifiche. Stesso dicasi per gli echo che fanno parte del gruppo.

**Reply by Scialla** [reply]
Confermo, per i gruppi il `tts` non funziona, devi usare `announce`
